src/scripts.js

```diff
@@ -171,7 +169,7 @@ function searchRecipes() {
   let searchInput = document.querySelector("#search-input");
   domUpdates.showAllRecipes(recipeData);
   let searchedRecipes = recipeData.filter(recipe => {
-    return recipe.name.toLowerCase().includes(searchInput.value.toLowerCase());
+    return recipe.name.toLowerCase().includes(searchInput.value.toLowerCase().trim());
```

Comment:
Nice sad path fix here! Looks like this makes it so that the user can't press the search button if it only has spaces in it.